.github/workflows/verify.yml

```diff
@@ -431,6 +431,59 @@ jobs:
         python3 -m pip install -r python/xnvme-cy-bindings/xnvme/cython_bindings/tests/requirements.txt --user
         python3 -m pytest --pyargs xnvme.cython_bindings -k 'test_version' -s
 
+    #
+    # Ramdisk testing
+    #
+    # skipping on distros with too old gcc/clang to build fio, and too old a Python to run cijoe
+    #
+    - name: Remove any existing cijoe installation and install from toolbox
+      if: (!(contains(matrix.container.ver, 'centos7') || contains(matrix.container.ver, 'opensuse') || contains(matrix.container.ver,
+        'buster') || contains(matrix.container.ver, 'bionic') || contains(matrix.container.ver, 'focal') || contains(matrix.container.ver,
+        '15.4') || contains(matrix.container.ver, '15.3') || contains(matrix.container.os, 'gentoo')))
```

Comment:
Could we define these once and store them in a variable? It's hard to tell if certain steps are meant for all or some of the containers.

Reply:
Perhaps, I haven't looked into that.

This PR was just a means to un-block:

- https://github.com/OpenMPDK/xNVMe/pull/128
- https://github.com/OpenMPDK/xNVMe/pull/144

Such that there was some kind of testing of the ramdisk. A lot more can be done, but that was out of scope here.